src/Symfony/Component/Mailer/Bridge/Sweego/composer.json

```
@@ -0,0 +1,37 @@
+{
+    "name": "symfony/sweego-mailer",
+    "type": "symfony-mailer-bridge",
+    "description": "Symfony Sweego Mailer Bridge",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Mathieu Santostefano",
+            "homepage": "https://github.com/welcoMattic"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=8.1",
+        "symfony/mailer": "^6.4|^7.0"
```

```suggestion
        "symfony/mailer": "^7.2"
```

otherwise the package is not included in the UnsupportedSchemeException